docs/quick_tutorial/authentication.rst

```diff
@@ -50,7 +50,9 @@ Steps
     :linenos:
 
 #. Create an ``authentication/tutorial/security.py`` module that can find our
-   user information by providing an *authentication policy callback*:
+   user information by providing an *authentication policy callback*.
```

Comment:
Undo changes to lines 53-55, and move them into the "Analysis" section where the code is explained in detail. At this point, the user should focus on code replication, then later analyze what they have written.

In the Analysis or See Also sections (as appropriate), include a link to the relevant Python docs for each call. For example, in reST:

```rest
:py:func:`hashlib.pbkdf2_hmac`
```

...will generate HTML that links to https://docs.python.org/3/library/hashlib.html#hashlib.pbkdf2_hmac